.gitignore

```diff
@@ -1,8 +1,33 @@
+
+# Created by https://www.gitignore.io/api/java,gradle
+
+### Java ###
+*.class
+
+# Mobile Tools for Java (J2ME)
+.mtj.tmp/
+
+# Package Files #
+*.jar
+*.war
+*.ear
+
+# virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
+hs_err_pid*
+
+
+### Gradle ###
 .gradle
-.idea
-build/**
-!build.gradle
-gradlew*
-gradle/**
-local.properties
-*iml
+/build/
```

Comment:
It ignores build/**, but it does not seem compatible with the document at https://git-scm.com/docs/gitignore.
The last item of pattern format. Therefore I prefer build/**.

What do you think?